src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -358,14 +358,19 @@ private JObject GenerateDependencies(JObject projectJsonRoot, ITaskItem[] extern
                 {
                     NuGetVersion nuGetVersion = NuGetVersion.Parse(dependency.GetMetadata("Version"));
                     PackageItem packageItem = new PackageItem(name, nuGetVersion);
+                    bool? forceVersion = dependency.GetMetadata("ForceVersion")?.Equals("true", StringComparison.OrdinalIgnoreCase);
```

Comment:
I wonder if it makes more sense to remove https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets#L132 and instead always use "Version" if it is present.